README.md

```diff
@@ -1,8 +0,0 @@
-The project title is Hello microverse
-Tech Stacks:
-    -Html
-    -Css
-Author:
- Name:Munana Clinton
- twitter:clintonmunana@gmail.com
- Github:https://github.com/clin2on3mun 
```

Comment:
- [ ] As requested by the previous reviewer kindly update the README file, please use your project name here and also remove the 2nd author as there is no 2nd author also your MIT license is broken please fix it
### Place to replace
![Annotation 2023-01-11 165220](https://user-images.githubusercontent.com/81727988/211791773-9888be2b-b4e9-4c16-9a66-ba8554c093ba.png)
![Annotation 2023-01-11 165133](https://user-images.githubusercontent.com/81727988/211791797-e225601b-a220-4d01-aa08-0e0685caef76.png)